src/styles/blocks/header.scss

```diff
@@ -38,5 +40,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
```

Comment:
The transition property should include what CSS properties it should apply to. For example, it could be 'transition: color 300ms;' to make it clear that the transition applies to the color property.